src/main/java/com/rtb/projectmanagementtool/init/InitTestData.java

```diff
@@ -0,0 +1,220 @@
+/** class initiates objects to test mvp on test servers */
+package com.rtb.projectmanagementtool.project;
+
+import com.google.appengine.api.datastore.*;
+import com.google.appengine.api.datastore.Query.*;
+import com.rtb.projectmanagementtool.auth.*;
+import com.rtb.projectmanagementtool.task.*;
+import com.rtb.projectmanagementtool.user.*;
+import com.rtb.projectmanagementtool.user.UserData.Skills;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/init-test-data")
+public class InitTestData extends HttpServlet {
+  // The users
+  private Long Sandy;
+  private Long Garry;
+  private Long Pearl;
+  private Long Patrick;
+
+  // The projects
+  private Long englishProject;
+  private Long csProject;
+  private Long historyProject;
+  private Long spanishProject;
+
+  // The tasks
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response)
+      throws ServletException, IOException {
+    System.out.println("Initiatializing test data");
+
+    // Authentication goes here
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+
+    // remove all entities at start
+    // this is mostly done so that the member variables of this
+    // class are initialized when the objects are created
+    removeAllEntities(datastore);
+
+    // Create UserDataObjects
+    creatUsers(datastore);
+
+    // Create ProjectDataObjects
+    creatProjects(datastore);
+
+    // Create TaskDataObjects
+    creatTasks(datastore);
+
+    // Load jsp for project page
+    response.sendRedirect("/logout");
+  }
+
+  public void removeAllEntities(DatastoreService datastore) {
+    removeAllEntities(datastore, new Query("User"));
+    removeAllEntities(datastore, new Query("Project"));
+    removeAllEntities(datastore, new Query("Task"));
+  }
+
+  public void removeAllEntities(DatastoreService datastore, Query query) {
+    PreparedQuery results = datastore.prepare(query);
+    for (Entity entity : results.asIterable()) {
+      datastore.delete(entity.getKey());
```

Comment:
You can batch delete by passing an array of keys I thought, instead of doing 1 delete per record.

I'm curious now if you can have a single ArrayList of keys combining the keys for users, projects, and tasks (since technically they're all the same object type) and pass that to delete.